spec/spec_helper.rb

```diff
@@ -1,22 +1,2 @@
-require 'rubygems'
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
-
-# COVERAGE=true rspec
-if ENV['COVERAGE']
-  require 'simplecov'
-  SimpleCov.start
-end
-
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'cnb'
-
-# Requires supporting ruby files with custom matchers and macros, etc,
-# in spec/support/ and its subdirectories.
-Dir[File.join('spec/support/**/*.rb')].each { |f| require f }
-
-RSpec.configure do |config|
-  config.expect_with :rspec do |c|
-    # disable the `should` syntax
-    c.syntax = :expect
-  end
-end
```

Comment:
hele tohle cely asi odstranovat nechceme, urcite chceme pouzivat expect misto should a taky to reportovat do toho CodeClimate. K tomu je myslim potreba ta coverage